packages/core-app-api/api-report.md

```diff
@@ -63,6 +66,17 @@ export class AlertApiForwarder implements AlertApi {
     post(alert: AlertMessage): void;
     }
 
+// @public
+export class AnalyticsApiForwarder implements AnalyticsApi {
+    // (undocumented)
+    event$(): Observable<ExtensionAwareAnalyticsEvent>;
```

Comment:
All app integrators will need to subscribe to this within a `useEffect` hook, so it might be worth exposing a `useAnalyticsSubscription` hook to do that automatically. That would also hide the observable-based implementation from the public api.

Reply:
There's also the possibility of using like https://github.com/streamich/react-use/blob/master/docs/useobservable MD which makes it a oneliner.

Reply:
There's a possibility too where we...  Don't actually expose it as observable, and instead require integrators to just implement `AnalyticsApi` (or instantiate a pre-built implementation from a plugin).

It's maybe less elegant for app integrators, but makes more sense, if you have a `GoogleAnalyticsApi` that actually just forwards things to GA without having to also fire off events to subscribers.